doc/index.rst

```diff
@@ -138,14 +141,14 @@ require changes to your existing code is logged in the :doc:`API changes
 <api/api_changes>` file.
 
 Toolkits
-========
+--------
```

Comment:
I preferred how it looked as level-1...

https://5220-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/index.html
versus
https://matplotlib.org

Reply:
Agreed, I think https://matplotlib.org/devdocs/ looks fine

Reply:
The issue that this is fixing the browser title which shows up as "Installation - Matplotlib 2.xxxxx documentation" vs "Matplotlib - Matplotlib 2.xxxx documentation".

I can seperate these two things, but we should fix the page title (as the browser sees it in `<head><title>Title<\title><\head>`)

Reply:
The title is now fixed by #10870.